### PR TITLE
Excluded 'libcephfs2','libcephfs-devel','libcephfs-dev'

### PR DESCRIPTION
**Pull request by yuriw**
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

### CODE REVIEW

**Comment by yuriw** [comment]
tests http://pulpito.front.sepia.ceph.com:80/yuriw-2016-11-21_17:50:59-upgrade:client-upgrade-master-distro-basic-smithi/

**Comment by yuriw** [comment]
tests http://pulpito.ceph.com/yuriw-2016-11-22_19:46:31-upgrade:client-upgrade-master-distro-basic-smithi/

http://tracker.ceph.com/issues/17809
http://tracker.ceph.com/issues/18001

**Comment by yuriw** [comment]
@jdurgin @dillaman Pls review and merge

File: suites/upgrade/client-upgrade/jewel-client-x/rbd/1-install/jewel-client-x.yaml [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
-    exclude_packages: ['ceph-mgr']
-- print: "**** done install infernalis"
+    branch: jewel
+    exclude_packages: ['ceph-mgr','libcephfs2','libcephfs-devel','libcephfs-dev']
+- print: "**** done install jewel"
 - install.upgrade:
    exclude_packages: ['ceph-test', 'ceph-test-dbg']
    client.1:
 - print: "**** done install.upgrade to -x on client.0"
-- ceph:
```
Comment (jdurgin) [inline suites/upgrade/client-upgrade/jewel-client-x/rbd/1-install/jewel-client-x.yaml]: looks like the ceph task is still needed here

Comment (yuriw) [inline suites/upgrade/client-upgrade/jewel-client-x/rbd/1-install/jewel-client-x.yaml]: Yes fixing

Comment (yuriw) [inline suites/upgrade/client-upgrade/jewel-client-x/rbd/1-install/jewel-client-x.yaml]: @jdurgin done

**Pull request merged**